x/qgb/types/keys.go

```diff
@@ -16,3 +22,29 @@ const (
 	// MemStoreKey defines the in-memory store key
 	MemStoreKey = "mem_payment"
 )
+
+var (
+	// ValsetConfirmKey indexes valset confirmations by nonce and the validator account address
+	// FIXME: For our keys, should they be `gravity1ahx7f8wyertuus9r20284ej0asrs085ceqtfnm` or `qgb1ahx7f8wyertuus9r20284ej0asrs085ceqtfnm`?
+	// i.e gravity1ahx7f8wyertuus9r20284ej0asrs085ceqtfnm
+	ValsetConfirmKey = "ValsetConfirmKey"
+)
+
+// GetValsetConfirmKey returns the following key format
+// prefix   nonce                    validator-address
+// [0x0][0 0 0 0 0 0 0 1][gravity1ahx7f8wyertuus9r20284ej0asrs085ceqtfnm]
+// FIXME: For our keys, should they be `gravity1ahx7f8wyertuus9r20284ej0asrs085ceqtfnm` or `qgb1ahx7f8wyertuus9r20284ej0asrs085ceqtfnm`?
```

Comment:
great question, I'll have to dig into this more. We might want to write an issue for it as we might want to record our findings. 

could it also be `qgb1ahx7f8wyertuus9r20284ej0asrs085ceqtfnm`?

Reply:
I still don't have much information about those values. I'll keep the FIXME and come back to them later if I don't stumble upon them when addressing the other parts.